tox.ini

```diff
@@ -10,13 +10,11 @@ envlist = py27, py34, py35, py36, docs
 setenv =
     PYTHONHASHSEED = 42
 commands =
-    python setup.py build_ext --inplace
-    py27,py34,py35: nosetests -v --with-coverage --cover-erase --cover-package=zarr zarr
+    py27,py34,py35: nosetests -v zarr
```

Comment:
So I missed this change the other day. Part of the problem with dropping coverage is we may miss Python 2 specific branches. Am suggesting we revert this one line in PR ( https://github.com/alimanfoo/zarr/pull/169 ). Though it would be good to know the motivation for this change in case there is a better solution.